src/app/host/[roomId]/components/RegisterFormProvider.tsx

```
@@ -0,0 +1,183 @@
+      case 'photos':
+        router.push(ROUTES.REGISTER_STEP(roomId).TITLE);
+        return;
+      case 'title':
+        const title = formData.get('title') as string;
+        updateData = { title };
+        break;
+      case 'description':
+        const description = formData.get('description') as string;
+        updateData = { description };
+        break;
+      case 'third':
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)